### PR TITLE
Add cluster-size section to system requirements (LGHP)

### PR DESCRIPTION
**Pull request by lproven**
I don't know why validation is failing, as it validated OK on my local machine.

### CODE REVIEW

File: xml/deployment_installation.xml [hunk 1]
```diff
@@ -55,6 +55,23 @@
...
+    <title>Cluster requirements</title>
+    <para>
+     &productname; only functions in a clustered configuration. It  requires a
+     group of physical or virtual machines in order to operate. The minimum
+     cluster size is four machines: one &admin_node;, one &kmaster;, and
+     two &kworker;s. This is in addition to separate machines providing 
+     storage.
+     </para>
+     <note>
+      <title>Test and proof-of-concept clusters</title>
```
Comment (tbazant) [inline xml/deployment_installation.xml]: Capitalize sentences in <title> markup

File: xml/deployment_installation.xml [hunk 2]
```diff
@@ -55,6 +55,23 @@
      </listitem>
     </itemizedlist>
    </sect3>
+   <sect3 xml:id="sec.caasp.installquick.clusreqs">
+    <title>Cluster requirements</title>
```
Comment (tbazant) [inline xml/deployment_installation.xml]: Capitalize sentences in <title> markup

**Comment by cwickert** [comment]
Now we have two entities:
&lt;!ENTITY master_node   "master node"&gt;
&lt;!ENTITY kmaster       "Master node"&gt;
This is everything but consistent. :-(

**Comment by lproven** [comment]
On Fri, 23 Mar 2018 06:36:29 -0700
Christoph Wickert <notifications@github.com> wrote:

> Now we have two entities:
> <!ENTITY master_node "master node">
> <!ENTITY kmaster "Master node">
> This is everything but consistent. :-(

That was me, although it was Tomáš's suggestion. I don't know why the
original entity for "Master node" has a capital letter in it, but I
guess that somewhere, it starts a sentence or something. I thought
that if I removed the capital letter, it might break something
somewhere. So I added a new entity without a capital M.

If that is not right, then what's the correct thing to do? Find where
the original use of the entity that needs a capital, and change it?

-- 
Liam Proven - Technical Writer, SUSE Linux s.r.o.
Corso II, Křižíkova 148/34, 186-00 Praha 8 - Karlín, Czechia
Email: lproven@suse.com - Office telephone: +420 284 241 084

**Comment by cwickert** [comment]
I'm not against new entities, but they should be consistent.

**Pull request merged**